gufe/tokenization.py

```diff
@@ -705,15 +705,23 @@ def from_json(cls, file: Optional[PathLike | TextIO] = None, content: Optional[s
             raise ValueError("Must specify either `content` and `file` for JSON input")
 
         if content is not None:
-            keyed_chain = json.loads(content, cls=JSON_HANDLER.decoder)
-            return cls.from_keyed_chain(keyed_chain=keyed_chain)
+            deserialized = json.loads(content, cls=JSON_HANDLER.decoder)
+            try:
+                return cls.from_keyed_chain(keyed_chain=deserialized)
+            except:
```

Comment:
Can we do a non-bare except here? I.e. is the error deterministic?

Reply:
Resolved!